client/src/components/Home.js

```diff
@@ -1,9 +1,11 @@
 import { useEffect, useState } from 'react';
 import styled from 'styled-components';
+import { HOST } from '../common';
+import AddChannel from './AddChannel';
 import DeleteButton from './DeleteButton';
 
 const Home = () => {
-  const [channels, setChannels] = useState([]);
+  let [channels, setChannels] = useState([]);
   const [isSubscribed, setIsSubscribed] = useState();
```

Comment:
once you make a change to `setChannels` that I mentioned in the other comment, you don't need this at all